processor/resourceattributetransposerprocessor/go.mod

```diff
@@ -4,8 +4,10 @@ go 1.18
 
 require (
 	github.com/stretchr/testify v1.8.1
-	go.opentelemetry.io/collector v0.64.1
-	go.opentelemetry.io/collector/pdata v0.64.1
+	go.opentelemetry.io/collector v0.66.0
+	go.opentelemetry.io/collector/component v0.65.0
+	go.opentelemetry.io/collector/consumer v0.65.0
```

Comment:
Should these be v0.66.0?

Reply:
yeah...I'll double check that. Not sure why it's 0.65.0.